packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -337,6 +337,24 @@ class RangeSlider extends StatefulWidget {
 
   @override
   _RangeSliderState createState() => _RangeSliderState();
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DoubleProperty('value1', values.start));
+    properties.add(DoubleProperty('value2', values.end));
+    properties.add(ObjectFlagProperty<ValueChanged<RangeValues>>('onChanged', onChanged, ifNull: 'disabled'));
+    properties.add(ObjectFlagProperty<ValueChanged<RangeValues>>.has('onChangeStart', onChangeStart));
+    properties.add(ObjectFlagProperty<ValueChanged<RangeValues>>.has('onChangeEnd', onChangeEnd));
+    properties.add(DoubleProperty('min', min));
+    properties.add(DoubleProperty('max', max));
+    properties.add(IntProperty('divisions', divisions));
+    properties.add(StringProperty('label1', labels.start));
```

Comment:
Although `values` can never be null, it looks like `labels` can be null. If `labels` is null and someone calls `debugFillProperties`, it will through an error.

To protect against this, change this to `labels?.start` and `labels?.end`.